docs/src/main/resources/templates/api.html

```diff
@@ -106,7 +106,7 @@ <h4>Request</h4>
   </ul>
 
   <p>
-    If the report must sent by email and the feature is enabled in the app, a <code>smtp</code> property
+    If the report must be sent by email and the feature is enabled in the app, an <code>smtp</code> property
```

Comment:
not a native tongue, but I think that `a` would be correct here, not `an` as suggested.

Reply:
Me neither. I was following this rule:
https://www.scribbr.com/commonly-confused-words/a-vs-an/

Reply:
Yes, but as this is an acronym and the “s” is pronounced əz, “an” should be used here:

> Acronyms beginning with a vowel sound use “an.”

(as described on https://www.scribbr.com/commonly-confused-words/a-vs-an/)